src/clr-angular/utils/drag-and-drop/providers/drag-event-listener.ts

```diff
@@ -61,8 +61,14 @@ export class ClrDragEventListener<T> {
     private customDragEvent(element: Node, startOnEvent: string, moveOnEvent: string, endOnEvent: string): () => void {
         let moveListener: () => void;
         let endListener: () => void;
+        let selecStartListener: () => void;
 
         return this.renderer.listen(element, startOnEvent, () => {
+            // This is needed to disable selection during dragging in IE11.
+            selecStartListener = this.renderer.listen("document", "selectstart", (selectEvent: Event) => {
+                selectEvent.preventDefault();
```

Comment:
Do we want to also stopPropagation() here?

Reply:
Yes. A select event shouldn't be triggered at all so I think it's appropriate to disable its propagation too. Also, just realized there is misspelling in "selecStartListener"